src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,16 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  if (!comments || comments.length === 0) {
```

Comment:
You don't need to render `CommentList` component at all if you don't have comments